pysc2/pysc2/lib/transform.js

```diff
@@ -0,0 +1,110 @@
+
+const path = require('path')
+const point = require(path.resolve(__dirname, './point.js'))
+
+class Transform extends object {
+	fwd_dist(dist){
+		throw NotImplementedError()
```

Comment:
where do these classes come from, NotImplementedError?

Reply:
It looks like it is from python built in exceptions. Should we implement it in js?